NICE.Identity.Management/ClientApp/src/components/ResultOrganisation/ResultOrganisation.tsx

```diff
@@ -2,7 +2,7 @@ import React, { FC } from "react";
 import { Link } from "react-router-dom";
 import Moment from "moment";
 import { Card } from "@nice-digital/nds-card";
-import { OrganisationType } from "../../models/types";
+import type { OrganisationType } from "src/models/types";
```

Comment:
I think 'type' goes inside the curly braces - not sure if it makes a difference for a single named import like this but would be clearer to read.... import {type OrganisationType} from "..."